composer.json

```
@@ -7,6 +7,7 @@
     "require": {
         "php": ">=7.2.0",
         "ext-json": "*",
+        "psr/log": "^1|^2|^3",
```

Let's keep the old/new style of the logical or in `composer.json` aligned.
The other versions utilize a double pipe.
```suggestion
        "psr/log": "^1 || ^2 || ^3",
```